rcsb/app/file/__init__.py

```diff
@@ -2,4 +2,4 @@
 __author__ = "John Westbrook"
 __email__ = "john.westbrook@rcsb.org"
 __license__ = "Apache 2.0"
-__version__ = "1.0"
+__version__ = "1.1"
```

Comment:
Thanks for remembering to do this. I'm wondering if we should change this to be 1.1.0 (Major.Minor.Patch) though, so that when small bug fixes are found we can increment the patch number instead of the minor.

Reply:
All right.